.github/workflows/build_and_test.yml

```diff
@@ -47,12 +47,13 @@ jobs:
                               ${{ matrix.use_s3_obj_store }} \
                               -DUSE_OPENTRACING=ON \
                               -DOMIT_TEST_OUTPUT=OFF\" "\
-              && script -q -e -c "make test"
+              && script -q -e -c "make test -DKEEP_APOLLO_LOGS=TRUE"
         - name: Prepare artifacts
           if: failure()
           run: |
             sudo chown -R $USER:$GROUP ${PWD}/build
             mv ${PWD}/build ${{ github.workspace }}/artifact/
```

Comment:
To reduce the bundle size, let's comment out this line as follows:
```
# Uncomment if core dump analysis is needed
# mv ${PWD}/build ${{ github.workspace }}/artifact/
```

Reply:
@teoparvanov , in this way we'll loose possibly useful core dumps.
I'd prefer to discuss it within an open forum with the rest of the team.